### ISSUE TITLE
HTML tag is missing `lang` attribute

### ISSUE THREAD

**Issue by JamieMagee** [sigstore/rekor-search-ui#54]
**Description**

Each `<html>` tag is required to have a `lang` attribute to meet WCAG 2.1 accessibility standards.

References:
- https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html
- https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang
